playground/mocks/data/api/v1/authn/forced-idp-discovery-linkedin-idp.json

```diff
@@ -0,0 +1,16 @@
+{
+  "subject": "okta:acct:undefined",
+  "links": [
+    {
+      "rel": "okta:idp",
+      "href": "http://localhost:3000/sso/idps/0oa4onxsxfUDwUb8u0g4?stateToken=00CazfnZW7EeVlIJoByKsUAdvMEcBvnN39We9ji5QL&login_hint=undefined#",
```

Comment:
we might wanna keep the `stateToken` value consistent across all mocks just so that its clear the same state token is getting passed in the flow.